package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@octokit/plugin-retry.js",
```

Comment:
can you remove the `.js` from the package name? We only use it in the repository names because there are other Octokits written in other languages.